src/Kernel-CodeModel/BlockClosure.class.st

```diff
@@ -384,6 +384,30 @@ BlockClosure >> method [
 	^ self compiledBlock
 ]
 
+{ #category : 'private timing' }
+BlockClosure >> microsecondClockValue [
+	"Answer the number of microseconds since Epoch"
+
+	^self primUTCMicrosecondsClock
```

Comment:
Why here we don't use `Time`?

Reply:
I thought that we want to get Time out of the kernel.
So I copied the primitive there.